chef/cookbooks/provisioner/files/default/crowbar_join.init.suse

```diff
@@ -4,7 +4,7 @@
 # Required-Start:    $syslog $network $remote_fs sshd
 # Should-Start:      neutron-ovs-cleanup xend libvirtd
 # Required-Stop:     $syslog $network $remote_fs sshd
-# Should-Stop:       openvswitch-switch ovs-usurp-config-br-fixed ovs-usurp-config-br-public
+# Should-Stop:       ovs-usurp-config-br-fixed ovs-usurp-config-br-public
```

Comment:
@jsuchome  "openvswitch-switch" was renamed to "openvswitch" already with SLES12-SP1. But anyway this change seems to be unneeded AFAICS "crowbar_join.init.suse " is only used on SLES11 anyway.